package.json

```diff
@@ -15,13 +15,13 @@
   "jsdelivr": "dist-browser/geotiff.js",
   "exports": {
     ".": {
-      "import": "./src/geotiff.js",
+      "import": "./dist-module/geotiff.js",
```

Comment:
This was an oversight on my part in #269, where I meant to make this change already.